tests/unit/dataactvalidator/test_scripts_loadFSRS.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+import pytest
+
+from dataactcore.models.fsrs import FSRSAward, FSRSSubaward
+from dataactvalidator.scripts import loadFSRS
+from tests.unit.dataactcore.factories import (
+    FSRSAwardFactory, FSRSSubawardFactory)
+
+
+@pytest.yield_fixture()
+def no_award_db(database):
```

Comment:
That said, I think it's important to use a real database when unit testing our validation rules. Outside of validation rules, though, do you think we should be mocking a database for the unit tests?